Podcast/SearchDiscoverViewController.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  SearchViewController.swift
+//  SearchDiscoverViewController.swift
```

Comment:
![image](https://user-images.githubusercontent.com/20246620/38657063-435578d0-3dec-11e8-9165-8293d2b325d1.png)
this gray line happens when you pop to root view controller (it fixes itself after a second or two)
this might be the navigation controller shadow image... 😢